cg/constants/constants.py

```diff
@@ -100,7 +100,7 @@ class SexOptions(StrEnum):
     UNKNOWN: str = "unknown"
 
 
-SARS_COV_REGEX = "^[0-9]{2}CS[0-9]{6}$"
+SARS_COV_REGEX = "^[0-9]{2}CS[0-9]{6}TEST$"
```

Comment:
```suggestion
SARS_COV_REGEX = "^[0-9]{2}CS[0-9]{6}$"
```

Reply:
just for testing.... easier to change the regex now so the test case can be picked up than to setup a new testing case with LIMS integration etc... we need to address this as this seems a bit weird of a check.